### PR TITLE
Add docs and tests

### PR DESCRIPTION
**Pull request by JSAssassin**
- Removed the skipped tests related to finding keystore since it looks like findKeystore() is not a thing in webkms-client.
- This PR will increase the test coverage to 87.72%%

### CODE REVIEW

**Comment by dmitrizagidulin** [comment]
Nicely done!

File: README.md [hunk 1]
```diff
@@ -1,2 +1,293 @@
...
+cd bedrock-kms-http
+npm install
+```
+
+## Usage
+
+In `lib/index.js` (or `main.js`, as appropriate):
+
+```js
+require('bedrock-kms-http');
```
Comment (dlongley) [inline README.md:39]: I think we should recommend ESM everywhere; CommonJS should be considered deprecated.

```suggestion
import 'bedrock-kms-http';
```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,293 @@
...
+```
+
```
Comment (dlongley) [inline README.md:41]:
```suggestion
```

File: test/mocha/30-operations-generate-key.js [hunk 3]
```diff
@@ -92,4 +98,50 @@ describe('generateKey with ipAllowList', () => {
     err.status.should.equal(403);
     err.data.type.should.equal('NotAllowedError');
   });
-}); // generateKey with ipAllowList
+}); // generateKey with ipAllowList∏∏
+
+describe('get public key description', () => {
+  it('gets public key decription', async () => {
```
Comment (dlongley) [inline test/mocha/30-operations-generate-key.js:104]:
```suggestion
  it('gets public key description', async () => {
```

**Review by dlongley — APPROVED** [review]
Thanks! I didn't do any checking of the various examples in the README other than a cursory glance through them.

File: test/mocha/30-operations-generate-key.js (hunk 3 above)
Comment (JSAssassin) [reply test/mocha/30-operations-generate-key.js]: Fixed cb856f2

**Review by dmitrizagidulin — APPROVED** [review]
👍

**Comment by mattcollier** [comment]
@JSAssassin please update to https://github.com/digitalbazaar/webkms-client/blob/main/CHANGELOG.md#800---2021-12-01

**Comment by JSAssassin** [comment]
> Do we need this new one? Would updating the name on the other one be better?

@dlongley Oh, my bad. I looked at the wrong coverage report and saw that the line wasn't being covered, so added it. I can remove it.

**Comment by dlongley** [comment]
I'm going to rebase and merge this.

**Pull request merged**